internal/clients/team1/client.go

```diff
@@ -32,3 +32,7 @@ func (c *client) GetID() int {
 func (c *client) Logf(format string, a ...interface{}) {
 	log.Printf("[%v]: %v", c.id, fmt.Sprintf(format, a...))
 }
+
+func (c *client) GetForageInvestment(gs common.GameState) int {

```

Comment:
this unfortunately is something we have to deal with in Golang.

In OOP languages like C++/C# we could have a base class implementing an interface and other classes inheriting from the base class. The base class could by default throw something like a `NotImplementedError`.

When we add stuff into the Client interface, we'd need to manually add in some code to replicate that behaviour; here, I put random numbers, but normally we'd return an error/default value that makes sense (e.g. 0 in this case)